src/languageservice/services/yamlHover.ts

```diff
@@ -141,7 +148,16 @@ export class YAMLHover {
           }
           result += `\`${toMarkdownCodeBlock(enumValue)}\`: ${markdownEnumValueDescription}`;
         }
-
+        if (markdownExamples.length !== 0) {
+          if (result.length > 0) {
+            result += '\n\n';
+          }
+          result += 'Examples: [\n\n';
```

Comment:
Not a big one but I am not sure if we need the `[` in here. The hovers looked better for me without them

Reply:
@gorkem I fixed hover format. I think it looks better. Thank you!!